settings.gradle

```diff
@@ -45,7 +45,7 @@ if (omnitureDependenciesAreSatisfied()) {
 }
 
 boolean freewheelDependenciesAreSatisfied() {
-    File dep = file("${System.getProperty("user.home")}/libs/AdManager.jar")
+    File dep = file("${System.getProperty("user.home")}/libs/AdManager.aar")
```

Comment:
@BCjwhisenant  I believe that if we're using `AdManager-6.28.0.aar` above, we should match the exact same string here too

Reply:
done